lib/Libs/Symfony/Yaml/function.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if (!function_exists('trigger_deprecation')) {
```

Comment:
the file is never used - should it be autoloaded?

Reply:
it is used in Inline.pp

Reply:
Used, yes, but is it loaded before being used?